example/dissim-plot/plot_kde.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import os, sys
+
+import matplotlib
+
+# Supress matplotlib display
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+
+import numpy as np
+from sklearn import neighbors
+from sklearn.externals import joblib
+
+sys.path.append("../../src")
+import handl
+import i_o
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--HANDL_scores', required=True)
+    parser.add_argument('-o', '--output', required=True)
+    parser.add_argument('--xmax', type=float, required=False, default=0.6)
+    return parser.parse_args()
+    
+
+def plot_and_save(scores_and_labels, xlabel, output,
+                  xmin=0.0, 
+                  xmax=0.6, 
```

Comment:
Should the `xmax` provided in the config file? Or is this usually the same across species pairs?

Reply:
xmax can be provided in the config file, 0.6 is just a value I picked as the default

Reply:
xmax can be provided via the configuration file. There is no obvious way to figure out what xmax should be a priori, so I just picked a default.